### DISCUSSION TITLE
It won't update the product information after changing it in the store

### DISCUSSION THREAD

**Discussion by davidsotaabdd** [hyochan/react-native-iap#1351] [Q&A]
As created by [InsideT](https://github.com/InsideT) here: https://github.com/dooboolab/react-native-iap/issues/1283

However, I'm using a _slightly_ older version of react-native-iap (all the rest is the same):

### Version of react-native-iap
4.6.3

### Version of react-native
0.63.3

### Platforms you faced the error (IOS or Android or both?)
Android

### Expected behavior
Update product information.

### Actual behavior
It won't update with the newest information of the product.

### Tested environment (Emulator? Real Device?)
Real Device

### Steps to reproduce the behavior

Retrieve the products and subscriptions

```
const skus = ["test_sku"];
let products = [];
const subscription = await RNIap.getSubscriptions(skus);
products = await RNIap.getProducts(skus);
products.push(subscription[0]);
```

Then change the information of your product.

**Comment by NounViraneath** [comment]
Hi, there
I ran into the same issue with my Android App using `"react-native-iap": "^7.0.5"` and `"react-native": "0.63.4"`
RNIap.getProducts(skus) won't return the latest products after I changed price or added new item.

I think, there might be some issue with Play Store Cache...
After I've tried to Clear Cache in Play Store App, RNIap.getProducts(skus) started to return the latest products correctly.
But RNIap.getProducts(skus) keeps returning cached products...

**Comment by oleksiy-nesterov** [comment]
The same situation for me... Maybe somebody knows some trick to force a product's data update?
